### ISSUE TITLE
No longer possible to skip-login or disable-settings-authorizer

### ISSUE THREAD

**Issue by maisam94** [kubernetes/dashboard#8854]
### What happened?

Steps to reproduce
I've added --enable-skip-login, --disable-settings-authorizer to the dashboard yaml files under API deployement.

Observed result
Deploy fail with unknown arguments --disable-settings-authorizer

### What did you expect to happen?

Expected result
Skip login and redirect to kubernetes dashboard

### How can we reproduce it (as minimally and precisely as possible)?

added --enable-skip-login, --disable-settings-authorizer

### Kubernetes Dashboard version

7.1.2

### Kubernetes version

1.28

**Comment by maciaszczykm** [comment]
Read release notes and docs. These args are not available anymore.

**Comment by maisam94** [comment]
But how can we skip-login and  disable-settings-authorizer??

**Comment by floreks** [comment]
Read [release notes](https://github.com/kubernetes/dashboard/releases/tag/kubernetes-dashboard-7.0.0).

TLDR: You can't skip the login screen. The only way is to use a reverse proxy that will add an authorization header with the predefined token. Settings now use user permissions for save action. We no longer allow using pod permissions.
